pennylane/templates/layers/simplified_two_design.py

```diff
@@ -0,0 +1,192 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Contains the ``SimplifiedTwoDesign`` template.
+"""
+# pylint: disable-msg=too-many-branches,too-many-arguments,protected-access
+from pennylane import numpy as np
+from pennylane.templates.decorator import template
+from pennylane.ops import CZ, RY
+from pennylane.templates import broadcast
+from pennylane.templates.utils import (
+    _check_shape,
+    _check_wires,
+    _check_number_of_layers,
+    _check_type,
+    _get_shape,
+)
+
+
+@template
+def entangler(par1, par2, wires):
+    """Implements a two qubit unitary consisting of a controlled-Z entangler and Pauli-Y rotations.
+
+    Args:
+         par1 (float or qml.Variable): parameter of first Pauli-Y rotation
+         par2 (float or qml.Variable): parameter of second Pauli-Y rotation
+         wires (list): two wire indices that unitary acts on
+    """
+
+    CZ(wires=wires)
+    RY(par1, wires=wires[0])
+    RY(par2, wires=wires[1])
+
+
+@template
+def SimplifiedTwoDesign(initial_layer, weights, wires):
+    r"""
+    Layers consisting of a simplified 2-design architecture of Pauli-Y rotations and controlled-Z entanglers
+    proposed in `Cerezo et al. (2020) <https://arxiv.org/abs/2001.00550>`_.
+
+    A 2-design is an ensemble of unitaries whose statistical properties are the same as sampling random unitaries
+    with respect to the Haar measure up to the first 2 moments.
+
+    The template is not a strict 2-design, since
+    it does not consist of universal 2-qubit gates as building blocks, but has been shown in
+    `Cerezo et al. (2020) <https://arxiv.org/abs/2001.00550>`_ to exhibit important properties to study "barren plateaus"
+    in quantum optimization landscapes.
+
+    The template starts with an initial layer of single qubit Pauli-Y rotations, before the main
+    :math:`L` layers are applied. The basic building block of the main layers are controlled-Z entanglers
+    followed by a pair of Pauli-Y gates.
```

Comment:
```suggestion
    followed by a Pauli-Y rotation gate on each wire.
```

Reply:
```suggestion
    followed by a pair of Pauli-Y gates (one for each wire).
```

Reply:
Right! I like the rewording, one thing that would be nice to include still is that these would be `qml.RY`s instead of `qml.PauliY`s :) (therefore added the word `rotation`, but I'm happy with any comparable rephrasing here).